tests/bindings/nodejs/HawkTracerClient.test.ts

```diff
@@ -77,7 +77,7 @@ test("start (resolve)", () => {
     expect.assertions(1);
     mockNativeClient.start = jest.fn().mockReturnValue(true);
     const client = new HawkTracerClient("abc");
-    return client.start().then(data => expect(data).toEqual(true));
+    return client.start().then(data => expect(data).toBeInstanceOf(HawkTracerClient));
```

Comment:
Better yet, you can check if it's the same instance as `client`.

Reply:
Good catch!